packages/gatsby-source-vtex/src/api.ts

```diff
@@ -1,7 +1,19 @@
+import type { Sort } from './types'
+
 export const api = {
   catalog: {
     category: {
       tree: (depth: number) => `/api/catalog_system/pub/category/tree/${depth}`,
+      search: ({
+        sort = 'OrderByTopSaleDESC',
```

Comment:
The default sort is `''` which has a different behaviour from `OrderByTopSaleDESC`